src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(
+  <div>
+
+  </div>,
```

Comment:
I like the idea of adding something basic here, what about adding a quick `<h1>Hello World!</h1>`?